### ISSUE TITLE
Can this play find my phone when lost?

### ISSUE THREAD

**Issue by MattL0** [grdn1337/node-red-contrib-apple-findme-reloaded#1]

**Comment by tb1337** [comment]
Currently not. What's the usecase? One can locate the phone by hitting a button on the Watch or on iCloud web?

**Comment by MattL0** [comment]
Hi,  thanks for the reply.

The usecase here would be to send via multiple automated way in my house , an mqtt message to nodered, so the phone rings and I can find it faster. 

Ex: pressing a zigbee switch button call an automation that send an mqtt message to nodered , so the node can hit the findmyphone end point and the phone rings even if it is in mute mode.

**Comment by tb1337** [comment]
Yes, could be an interesting scenario. I will take a look at it and add a second node.

**Comment by MattL0** [comment]
Thanks

**Comment by regissg** [comment]
I am interested as well to be able to activate the findmyiphone feature. My use case is to be able to ask Alexa through node red to find my iphone
